File: src/domain/constant/XmlNodeName.java
package domain.constant;

/**
 *
 */
public enum XmlNodeName {
    PROPERTIES("properties"), DEPENDENCY("dependency"), VERSION("version"),
    GROUP_ID("groupId"), ARTIFACT_ID("artifactId"), PROJECT("project");

    private final String xmlName;

    XmlNodeName(final String xmlName) {
        this.xmlName = xmlName;
    }

    /**
     *
     * @return
     */
    public String getXmlName() {
        return xmlName;
    }
}

File: src/main/kotlin/com/mirkoalicastro/mavenversionrefactor/domain/constant/XmlNodeName.kt
package com.mirkoalicastro.mavenversionrefactor.domain.constant

enum class XmlNodeName(val xmlName: String) {
    PROPERTIES("properties"), DEPENDENCY("dependency"),
    PLUGIN("plugin"), VERSION("version"),
    GROUP_ID("groupId"), ARTIFACT_ID("artifactId"),
    PROJECT("project")
}
